mmv1/templates/terraform/examples/privileged_access_manager_entitlement_basic.tf.erb

```diff
@@ -0,0 +1,38 @@
+resource "google_privileged_access_manager_entitlement" "<%= ctx[:primary_resource_id] %>" {
+    provider =  google-beta
+    entitlement_id = "<%= ctx[:vars]['entitlement_id'] %>"
+    location = "global"
+    max_request_duration = "43200s"
+    parent = "projects/itsvarsharma-pam-testing"
```

Comment:
Hi Nick, we can use the same project that is used to run tests. But we need to "setup" privileged access manager manually on that project (Setup here means calling an API that will create a p4sa and grant some permissions to it. This is an additional step to enabling the service in that project). 
Are the automated tests run using static projects? If so, is there a human user with IAM Admin permission to setup PAM in that project?
Also, we need at least one valid IAM principal to use in tests (see lines #11 and #33). Can we use a normal user email here? Or do we need to create some accounts for testing?
We prefer not writing handwritten tests. Thanks

Reply:
> handwritten 

These were already made for the update tests, so it wouldnt be too major an update from there. But regardless, from how the situation sounds I don't think it would be necessary: the resource once in public preview (soft launch or otherwise) would work without these conditions correct?